docusaurus/docs/user-docs/intro.md

```diff
@@ -5,109 +5,50 @@ sidebar_label: Welcome!
 
-| Profile & Experience  | Instructions                                                                                            |
-| --------------------- | ------------------------------------------------------------------------------------------------------- |
-| First name            | Write your first name in the textbox.                                                                   |
-| Last name             | Write your last name in the textbox.                                                                    |
-| Email                 | Write your complete email address in the textbox.                                                       |
-| Username              | (optional) Write a username in the textbox.                                                             |
-| Interface language    | Among the drop-down list, choose a language for your Strapi application interface.                      |
-| Interface mode        | Among the drop-down list, choose a mode for your Strapi application interface: either "Light mode" or "Dark mode". Note that by default, the chosen mode for a Strapi application is based on the browser's mode. |
+## What you will find here
 
-4. Click on the **Save** button.
+The table of content in this User Guide displays 7 main sections.
 
-### Changing your password
+Clicking on any of the following cards will direct you to the introductory page for that section, with additional details and concepts:
 
-To change the password of your account:
-
-1. Go to your administrator profile.
-2. Fill in the password-related options:
-
-| Password modification |                                                                                                         |
-| --------------------- | ------------------------------------------------------------------------------------------------------- |
-| Current password      | Write your current password in the textbox. <br/> 💡 You can click on the eye icon for the password to be shown. |
-| Password              | Write the new password in the textbox. <br/> 💡 You can click on the eye icon for the password to be shown.      |
-| Password confirmation | Write the same new password in the textbox. <br/> 💡 You can click on the eye icon for the password to be shown. |
-
-3. Click on the **Save** button.
-
-
----
-
-Congratulations on being a new Strapi user! You're now ready to discover all the features and options that Strapi has to offer!
+<CustomDocCardsWrapper>
+  <CustomDocCard emoji="📝" title="Content Manager" description="Manage and publish all content types created with the Content-type Builder." link="/user-docs/content-manager" />
+  <CustomDocCard emoji="📚" title="Content Type Builder" description="Create, Manage and Publish Content Types including collection types." link="/user-docs/content-type-builder" />
```

Comment:
There are 2 issues that you need to fix on this card's description:
- Style-related: capitalization and punctuation: it should be written 'Create, manage, and publish' (no uppercase on words inside a sentence except if they are Strapi features, plus usage of the [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/)
- Content-related: "Create, manage and publish content-types…" are you sure this is what the Content-Types Builder is about? 😄  I'm sure you can come up with a better definition of the CTB 🙂